src/logger.js

```diff
@@ -1,29 +1,68 @@
 import { TracerGlobals } from './globals';
-
+import { isStoreLogs } from './utils';
 const LOG_PREFIX = '#LUMIGO#';
 const WARN_CLIENT_PREFIX = 'Lumigo Warning';
 
+const MAX_DUPLICATE_LOGS = 50;
+
+export const LogStore = (() => {
+  let logSet = new Set([]);
+  let duplicateLogsCount = 0;
+
+  const addLog = (type, message, object) => {
+    const logObj = JSON.stringify({ type, message, object });
+    if (!logSet.has(logObj)) {
+      logSet.add(logObj);
+    } else {
+      duplicateLogsCount++;
+    }
+    isEmergencyMode() && printLogs();
```

Comment:
I think that our convention is using `if isEmergencyMode()`. Isn't it?

Reply:
Only on frontend maybe, here most of the 1 line ifs it's like that